include/klee/Expr/Assignment.h

```diff
@@ -27,45 +28,44 @@ using symcretes_ty = SymcreteOrderedSet;
 
 class Assignment {
 public:
-  typedef std::map<const Array *, SparseStorage<unsigned char>> bindings_ty;
+  using bindings_ty = ImmutableMap<const Array *, SparseStorage<unsigned char>>;
```

Comment:
You can use `PersistentMap` instead of `ImmutableMap`, since the persistent version is a wrapper for convenience